topology/probes/k8s/graph.go

```diff
@@ -65,32 +66,48 @@ func addMetadata(g *graph.Graph, n *graph.Node, details interface{}) {
 func newEdgeMetadata() graph.Metadata {
 	m := graph.Metadata{
 		"Manager":      managerValue,
-		"RelationType": "Association",
+		"RelationType": "association",
 	}
 	return m
 }
 
+func dumpGraphLink(parent, child *graph.Node) string {
+	return fmt.Sprintf("%s -> %s", dumpGraphNode(parent), dumpGraphNode(child))
+}
+
 func addLink(g *graph.Graph, parent, child *graph.Node) *graph.Edge {
 	m := newEdgeMetadata()
 	if e := g.GetFirstLink(parent, child, m); e != nil {
 		return e
 	}
+
+	logging.GetLogger().Debugf("Adding link: %s", dumpGraphLink(parent, child))
 	return g.Link(parent, child, m, hostID)
 }
 
+func delLink(g *graph.Graph, parent, child *graph.Node) {
```

Comment:
I prefer to modify Unlink so that it accepts Metadata as an additional parameter

Reply:
Ok

Reply:
This `delLink() interface is consistent with `addLink(..)` both assume RelationType == "association", for ownership you have `addOwnershipLink()` and `delOwnershipLink()` is not needed so it was not implemented.